app/templates/public/index.hbs

```diff
@@ -1,3 +1,6 @@
 <div class="event description">
   {{sanitize model.event.description}}
-</div>
\ No newline at end of file
+  <div class="row">
```

Comment:
Keep row outside the `event description` div. Put it inside a a div with classes `event speakers`